spec/nio/selectables/udp_socket_spec.rb

```diff
@@ -22,7 +22,17 @@
   end
 
   let :writable_subject do
-    pending "come up with a writable UDPSocket example"
```

Comment:
Can you see what the error is on the rescue?  I couldn't fill up the UDP socket (or generate an error) on Windows.

I'm confused why just two of the travis build jobs failed, and both state 'Expected pending "UDPSocket Error' to fail. No error was raised."

What if the end while condition is removed, and keep some kind of rescue to make sure it can be written to.  Maybe that will be better?